components/kyma-environment-broker/internal/storage/dbsession/read.go

```diff
@@ -0,0 +1,30 @@
+package dbsession
+
+import (
+	dbr "github.com/gocraft/dbr"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/storage/dberr"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/storage/schema"
+)
+
+type readSession struct {
+	session *dbr.Session
+}
+
+func (r readSession) GetInstanceByID(instanceID string) (internal.Instance, dberr.Error) {
+	var instance internal.Instance
+
+	err := r.session.
+		Select("*").
+		From(schema.InstancesTableName).
+		Where(dbr.Eq(schema.InstancesTableName+".instance_id", instanceID)).
```

Comment:
IMO table name is not neccessary 
see https://godoc.org/github.com/gocraft/dbr#Eq